pkg/reconciler/instances/ory/action.go

```diff
@@ -194,15 +201,12 @@ func (a *preReconcileAction) updateSecret(ctx context.Context, client kubernetes
 	return err
 }
 
-func (a *preReconcileAction) rolloutHydraDeployment(ctx context.Context, client kubernetes.Interface, logger *zap.SugaredLogger) error {
-	data := fmt.Sprintf(`{"spec":{"template":{"metadata":{"annotations":{"kubectl.kubernetes.io/restartedAt":"%s"}}}}}`, time.Now().String())
-
-	_, err := client.AppsV1().Deployments("kyma-system").Patch(ctx, "ory-hydra", types.StrategicMergePatchType, []byte(data), metav1.PatchOptions{})
-
+func (a *postReconcileAction) rolloutHydraDeployment(ctx context.Context, client internalKubernetes.Client, deployment string, logger *zap.SugaredLogger) error {
+	err := a.rolloutHandler.RolloutAndWaitForDeployment(ctx, deployment, oryNamespace, client, logger)
 	if err != nil {
-		return errors.Wrap(err, "Failed to rollout ory hydra")
+		return errors.Wrapf(err, "Failed to rollout %s deployment", deployment)
 	}
-	logger.Info("ory-hydra restarted")
+	logger.Infof("%s deployment restarted", deployment)
```

Comment:
We used the phrase `Failed to rollout` in a negative scenario. How about sticking to it? In this case you could call it: `(...) deployment rolled out` or to be fully compliant: `Performed rollout restart of deployment (...)`.